src/pages/CirclesPage/index.ts

```diff
@@ -0,0 +1,2 @@
+import CirclesPage from './CirclesPage';
```

Comment:
I believe I mentioned in our last call that we should rename it to CirclesPage; but if you haven't started on renaming already, let's just leave it as AdminPage, for the sake of creating less conflict with the main branch.

Reply:
You can still have a CirclesPage folder with a top-level component as I described in my last comment that imports files from AdminPage/.

Reply:
It's ok to apply the new design changes to the original AdminPage component, e.g. moving the buttons, making the table full-width, & adding a gray background. 

The one change you'll want to make to keep the component backwards-compatible is to move the Add Circle button, since the original page has no organization header:

![Coordinape](https://user-images.githubusercontent.com/193667/139770090-53028d55-624d-4d0b-b2ec-a4b1cb93147f.png)

Reply:
This is how I'm envisioning the component reuse, in case it's not clear:
![Screen Shot 2021-11-01 at 6 09 33 PM](https://user-images.githubusercontent.com/193667/139770705-cd67ec0c-3772-4bd6-a97d-afcc05112b45.png)